redis.conf

```diff
@@ -1333,10 +1333,10 @@ appendonly no
 # The base name of the append only file.
 # 
 # Redis can persist the data to a set of append-only files, they are divided
-# into two types, one is the the base type, it represents an initial (RDB
-# serialized) snapshot of the data present when the AOF is rewritten. The others
-# are incremental type, they contain the incremental commands since the last AOF
-# rewrite.
+# into two types, one is the the BASE type, it represents an initial (RDB
+# or AOF format) snapshot of the data present when the AOF is rewritten. The 
```

Comment:
Try to avoid using the term "rewrite" before explaining it.
```suggestion
# or AOF format) snapshot of the data present when the AOF is initially created (or rewritten). The 
```

Reply:
the AOF initialization don't create base AOF.

Reply:
it will soon (see #9794).
but also, considering the state of this PR, isn't it better to tag the first AOF file we create as "base"?
i.e. when loading an aof manifest, the first file we load must always be a "base" file.
i.e. it would be wrong to start from an "incremental" file since it implies we're starting from the middle with no view of the history, so in that sense i think that on empty startup, we should change the code and tag the first file as "base"

Reply:
Maybe we'll change that text in the next PR.